src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -61,6 +69,12 @@
             :default-message="defaultMessage"
             :api-error="error"
         ></error-info>
+        <workflow-report-log-dialog
+            id="report-log-dialog"
+            :report-key=showLogForReportKey
+            :workflow-key=selectedWorkflowKey
+            @close="closeReportLogDialog"
+        ></workflow-report-log-dialog>
```

Comment:
```suggestion
       </workflow-report-log-dialog>
```

Formttting here is a bit weird and in general in this template; could we tidy up the right arrow braces where they've flowed onto the next line?